docker-compose/obam-obkm-with-obbi/docker-compose.yml

```diff
@@ -0,0 +1,145 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+
+version: '3.7'
+services:
+  
+  mysql:
+    container_name: mysql
+    image: mysql:5.7.24
+    ports:
+      - "3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+    volumes:
+      - "./mysql/scripts/berlin:/docker-entrypoint-initdb.d"
+      - "./mysql/scripts/my.cnf:/etc/mysql/my.cnf"
+    ulimits:
+      nofile:
+        soft: 20000
+        hard: 40000
+    command: [--ssl=0]
+    healthcheck:
+      test: ["CMD", "mysqladmin" ,"ping", "-uroot", "-proot"]
+      interval: 10s
+      timeout: 10s
+      retries: 5
+      start_period: 30s
+
+  obbi-worker:
+    container_name: obbi-worker
+    image: docker.wso2.com/wso2-obbi-worker
+    healthcheck:
+      test: ["CMD", "nc", "-z", "localhost", "9444"]
+      interval: 10s
+      timeout: 10s
+      start_period: 150s
+      retries: 5 
+    depends_on:
+      - "mysql"
+    volumes:
+      - "./obbi:/home/wso2carbon/wso2-config-volume"
+      - "./wait-for-it.sh:/home/wso2carbon/wait-for-it.sh"
+    ports:
+      - "9444:9444"
+      - "8006:8006"
+      - "8007:8007"
+    entrypoint:
+      - "/home/wso2carbon/wait-for-it.sh"
+      - "mysql:3306"
+      - "-t"
+      - "240"
+      - "--"
+      - "./docker-entrypoint.sh"
+
+  obbi-dashboard:
+    container_name: obbi-dashboard
+    image: docker.wso2.com/wso2-obbi-dashboard
+    healthcheck:
+      test: ["CMD", "nc", "-z", "localhost", "	9449"]
```

Comment:
@kaumini please remove the unnecessary spaces in the port specification `9449`.

Reply:
Fixed with 3456e7336ea24f8826ba640c54a6b1d9e94154b1